op-proposer/proposer/service.go

```diff
@@ -249,6 +249,7 @@ func (ps *ProposerService) initRPCServer(cfg *CLIConfig) error {
 	if cfg.RPCConfig.EnableAdmin {
 		adminAPI := rpc.NewAdminAPI(ps.driver, ps.Metrics, ps.Log)
 		server.AddAPI(rpc.GetAdminAPI(adminAPI))
+		server.AddAPI(txmgr.NewTxmgrApi(ps.TxManager, ps.Metrics, ps.Log))
```

Comment:
Currently the `op-challenger` does not have an rpc server. Should we create one (similar to how the `op-proposer` has an rpcServer implemented here) so that we also add the `txmgr_` (and perhaps `admin_`) rpc methods to that service?

Reply:
I don't recall ever looking at adding RPC to the op-challenger. Certainly happy to have it added since it seems useful and if someone can take on that work. Otherwise we should log an issue to track it but I don't think the proofs team will have capacity for it in the short term.